### DISCUSSION TITLE
reactlog placement

### DISCUSSION THREAD

**Discussion by michaeld96** [Appsilon/rhino#314] [Q&A] [answered]
Hi!

I was wondering, where would be the best place to put `reactlog()` in a rhino application?

**Comment by kamilzyla** [comment, marked as answer]
Call [`reactlog_enable()`](https://rstudio.github.io/reactlog/reference/setReactLog.html) in your `app/main.R` (outside of any functions, e.g. just below `box::use()`). Run [`shiny::reactlogShow()`](https://rdrr.io/pkg/shiny/man/reactlog.html) from the R console to view the results.
